src/EditAnnotations.cpp

```diff
@@ -1535,26 +1549,40 @@ Annotation* EngineMupdfCreateAnnotation(EngineBase* engine, AnnotationType typ,
 
     pdf_update_annot(ctx, annot);
     auto res = MakeAnnotationPdf(epdf, annot, pageNo);
+
+    std::optional <PdfColor> col;

```

Comment:
We don't use std::, please use ColorUnset instead

Reply:
Fixed.